web/pages/_app.tsx

```diff
@@ -2,9 +2,10 @@ import { useEffect } from 'react';
 import { getCookieConsentValue } from 'react-cookie-consent';
 import TagManager from 'react-gtm-module';
 import CookieConsent from '../components/CookieConsent';
+import Nav from '../components/Nav';
 import '../styles/globals.css';
 
-function MyApp({ Component, pageProps }) {
+function MyApp({ Component, pageProps, router }) {
```

Comment:
```suggestion
function MyApp({ Component, pageProps }) {
```

Reply:
Ah yes, thanks… I accidentally committed some experimental code. Missed this part.